### PR TITLE
Minor function update

### LINKED ISSUE #118: About the maximum length validation of the values in the segment table's rule column
**Is your feature request related to a problem? Please describe.**
I need to group the String type userId into a crowd group, where there are about 400000 userIds, each userId length is 7 characters, I try to put 1500000 userIds into the array, after testing, the rule engine parsing error
As an example
```
[
    {
        "conditions": [
            {
                "type": "string",
                "subject": "userId",
                "predicate": "is one of",
                "objects": [
                    "3292",
                    "3293"...andothers
                ]
            }
        ],
        "name": "rule1"
    }
]
```

**Describe alternatives you've considered**
Based on my experience I guess the character length exceeds the limit of the rules engine

So I will userId every 100000 group, divided into 4 groups, after testing, the rules engine can be normal parsing

```
[
    {
        "conditions": [
            {
                "type": "string",
                "subject": "userId",
                "predicate": "is one of",
                "objects": [
                    "3292",
                    "3293"...and others
                ]
            }
        ],
        "name": "rule1"
    },{
        "conditions": [
            {
                "type": "string",
                "subject": "userId",
                "predicate": "is one of",
                "objects": [
                    "103292",
                    "103293"...and others
                ]
            }
        ],
        "name": "rule2"
    },{
        "conditions": [
            {
                "type": "string",
                "subject": "userId",
                "predicate": "is one of",
                "objects": [
                    "203292",
                    "203293"...and others
                ]
            }
        ],
        "name": "rule3"
    },{
        "conditions": [
            {
                "type": "string",
                "subject": "userId",
                "predicate": "is one of",
                "objects": [
                    "303292",
                    "303293"...and others
                ]
            }
        ],
        "name": "rule4"
    }
]
```

**Additional context**
the size of `rule` column that i got is 3.07 MB 

 i sent a request and wating reponse costed 70ms with single userId

### PR DESCRIPTION
**Pull request by panchongyang**
Fix #118  .

Changes proposed in this pull request:

* Big size of toogle have tips

### CODE REVIEW

File: ui/src/pages/segmentEdit/components/Info/index.tsx [hunk 1]
```diff
@@ -153,6 +157,28 @@ const Info = () => {
...
+    if (size >= MAX_SIZE) {
+      state = 'large';
+    } else if (size > MAX_SIZE * 0.8) {
+      state = 'near';
+    }
+    return state;
+  }, [size]);
+
+  useEffect(() => {
+    console.log(sizeState, size, MAX_SIZE - size);
```
Comment (frankzhaopeng) [inline ui/src/pages/segmentEdit/components/Info/index.tsx:179]: Delete console.log

File: ui/src/locales/en-US.json [hunk 2]
```diff
@@ -113,6 +113,7 @@
   "common.diff.tips": "The following is a before and after comparison of this change.",
   "common.flow.explain": "Explain",
   "common.know.text": "I see",
+  "common.know.not.me.text": "Got it",
```
Comment (frankzhaopeng) [inline ui/src/locales/en-US.json:116]: Why not use I see?

File: ui/src/locales/en-US.json [hunk 3]
```diff
@@ -341,6 +342,8 @@
   "targeting.semver.explain": "What is semver?",
   "targeting.semver.more": "Learn more",
   "targeting.rule.not.empty": "Please add a rule condition",
+  "targeting.size.tips": "The total content of this page can not exceed 1 M, and will not be released normally when it is larger than 1 M. Currently, it has reached {size} KB!",
```
Comment (frankzhaopeng) [inline ui/src/locales/en-US.json:345]: 1 MB?

File: ui/src/locales/en-US.json [hunk 4]
```diff
@@ -341,6 +342,8 @@
...
+  "targeting.size.tips.title": "Warning Tips",
```
Comment (frankzhaopeng) [inline ui/src/locales/en-US.json:346]: Warning tips

File: ui/src/pages/segmentEdit/components/Info/index.tsx [hunk 5]
```diff
@@ -26,6 +26,8 @@ import { SEGMENT_EDIT_PATH } from 'router/routes';
 import { IRule, ICondition } from 'interfaces/targeting';
 import { IVersionParams } from 'interfaces/project';
 import styles from './index.module.scss';
+import { getBytes, MAX_SIZE } from 'utils/bytes';
+import SizeTips from 'components/SizeTips';
```
Comment (frankzhaopeng) [inline ui/src/pages/segmentEdit/components/Info/index.tsx:30]: change the import position

File: ui/src/utils/bytes.ts [hunk 6]
```diff
@@ -0,0 +1,10 @@
+export function getBytes(str: string) {
+  let bytes = str.length,
+    i = 0;
+  for (; i < bytes; i++) if (str.charCodeAt(i) > 255) bytes++;
+
+  return bytes;
+}
+
+export const MB = 1048576;
```
Comment (frankzhaopeng) [inline ui/src/utils/bytes.ts:9]: If constant `MB` is useful?

File: ui/src/locales/en-US.json (hunk 2 above)
Comment (panchongyang) [reply ui/src/locales/en-US.json:116]: A little difference，can be changed to “I  See”

File: ui/src/locales/en-US.json (hunk 3 above)
Comment (panchongyang) [reply ui/src/locales/en-US.json:345]: ok

**Pull request merged**